camera_calibration/src/camera_calibration/calibrator.py

```diff
@@ -1250,6 +1260,13 @@ def report(self):
         print("self.T =", numpy.ravel(self.T).tolist())
         print("self.R =", numpy.ravel(self.R).tolist())
 
+        r = PyKDL.Rotation(self.R[0][0], self.R[0][1], self.R[0][2],
+                           self.R[1][0], self.R[1][1], self.R[1][2],
+                           self.R[2][0], self.R[2][1], self.R[2][2])
+        return("Extrinsic transformation from the right camera to the left camera:\nT = "
+                +str(numpy.ravel(self.T).tolist())+"\nR = "+str(numpy.ravel(self.R).tolist())
+                +"\nquaternion = "+str(list(r.GetQuaternion()))+"\nRPY = "+str(list(r.GetRPY())))
```

Comment:
missing whitespaces after return and around the + signs.
it might make sense to switch to f-strings here to make the resulting string easier to read/understand